tests/ResultPrinter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yiiunit;
+
+/**
+ * IsOneOfAssert asserts that the value is one of the expected values.
+ */
+class ResultPrinter extends \PHPUnit\TextUI\ResultPrinter
```

Comment:
Using custom `ResultPrinter` as a solution, does not seem right.
What if some developer creates component or extension, which operates session and wishes to write a unit tests for it? He will have to create his own `ResultPrinter` as well, which does not feel right.

We should ensure framework testability.

Reply:
+1
I must been redefine PHPStorm printer - his realization send headers too.
```php
class PhpStorm_Codeception_ReportPrinter_Redefine extends PHPUnit_TextUI_ResultPrinter
{
    /**
     * @inheritDoc
     */
    public function __construct(
        $out = null,
        $verbose = false,
        $colors = self::COLOR_DEFAULT,
        $debug = false,
        $numberOfColumns = 80,
        $reverse = false
    ) {
        parent::__construct(STDOUT, $verbose, $colors, $debug, $numberOfColumns, $reverse);
    }

    /**
     * Flush buffer and close output if it's not to a STDOUT stream
     */
    public function flush()
    {
        if ($this->out != STDOUT) {
            parent::flush();
        }
    }
}

class PhpStorm_Codeception_ReportPrinter extends PhpStorm_Codeception_ReportPrinter_Redefine
...
```

Reply:
It's not a final solution, I'm just trying to make it work somehow :)
There is an issue in PHPUnit repo https://github.com/sebastianbergmann/phpunit/issues/1193

Reply:
@SilverFire yes, I tell about this problem. p.s.: Don't forgot `flush()`

Reply:
Done